emperor/support_files/js/model.js

```diff
@@ -1,14 +1,97 @@
+/**
+ *
+ * @author Jamie Morton, Jose Navas Molina, Andrew Hodges & Yoshiki
+ *         Vazquez-Baeza
+ * @copyright Copyright 2013--, The Emperor Project
+ * @credits Jamie Morton, Jose Navas Molina, Andrew Hodges & Yoshiki
+ *          Vazquez-Baeza
+ * @license BSD
+ * @version 0.9.51-dev
+ * @maintainer Yoshiki Vazquez Baeza
+ * @email yoshiki89@gmail.com
+ * @status Development
+ *
+ */
+
+
+/**
+ *
+ * @name Plottable
+ *
+ * @class Represents a sample and the associated metadata in the ordination
+ * space.
+ *
+ */
+
 /* Represents an object that can be drawn (i.e. a sample or taxa)
```

Comment:
I think we can remove this comment, as it is outdated. Your comment is more extensive.